clang/lib/Driver/Driver.cpp

```diff
@@ -3217,9 +3234,11 @@ class OffloadingActionBuilder final {
 
       Arg *SYCLLinkTargets = Args.getLastArg(
                                   options::OPT_fsycl_link_targets_EQ);
-      CompileDeviceOnly = SYCLLinkTargets &&
-                          SYCLLinkTargets->getOption().matches(
-                              options::OPT_fsycl_link_targets_EQ);
+      WrapDeviceBinary = Args.hasArg(options::OPT_fsycl_link);
```

Comment:
I assume `WrapDeviceBinary ` means that we only wrap device code by the offload wrapper, If this is right, maybe we should mention that we wrap **only** device code?
For example change `WrapDeviceBinary` to something like `WrapDeviceOnlyBinary` .

Reply:
That's right, this is only wrapping the device code.  I will update the variable name to better represent expectations.

Reply:
Done